tests/commons.py

```diff
@@ -108,13 +108,13 @@ def change_redis_client_status(activated: bool) -> None:
 
 async def create_user_with_groups(
     groups: list[GroupType],
+    account_type: AccountType = AccountType.external,
```

Comment:
I think by default we could create student users. This corresponds to the most common use case.